db/migrations/migrations.go

```diff
@@ -74,65 +74,69 @@ func migrate(db *gorm.DB) error {
-					--
-					SELECT create_hypertable('ts_traceroute_results', by_range('time'));`).Error
+                    SELECT create_hypertable('ts_host_runtime_stats', by_range('time'));
+                    --
+                    SELECT create_hypertable('ts_dns_results', by_range('time'));
+                    SELECT create_hypertable('ts_dns_results_answer', by_range('time'));
+                    --
+                    SELECT create_hypertable('ts_http_results', by_range('time'));
+                    --
+                    SELECT create_hypertable('ts_icmp_results', by_range('time'));
+                    --
+                    SELECT create_hypertable('ts_traceroute_results', by_range('time'));
+                    SELECT create_hypertable('ts_traceroute_results_hop', by_range('time'));`).Error
 				if err != nil {
 					return err
 				}
 
 				// indices
 				err = tx.Exec(`
-					CREATE INDEX idx_runtime_sensor_time ON ts_host_runtime_stats (sensor_id, time DESC);
-					--
-					CREATE INDEX idx_dns_results_sensor_time ON ts_dns_results (sensor_id, time DESC);
-					CREATE INDEX idx_dns_results_answer_sensor_time ON ts_dns_results_answer (sensor_id, time DESC);
-					-- CREATE INDEX idx_dns_results_task ON ts_dns_results (task_id);
-					-- CREATE INDEX idx_dns_results_answer_task ON ts_dns_results_answer (task_id);
-					--
-					CREATE INDEX idx_http_results_sensor_time ON ts_http_results (sensor_id, time DESC);
-					--
-					CREATE INDEX idx_icmp_results_sensor_time ON ts_icmp_results (sensor_id, time DESC);
-					--
-					CREATE INDEX idx_traceroute_results_sensor_time ON ts_traceroute_results (sensor_id, time DESC);`).Error
+                    CREATE INDEX idx_runtime_sensor_time ON ts_host_runtime_stats (sensor_id, time DESC);
+                    --
+                    CREATE INDEX idx_dns_results_sensor_time ON ts_dns_results (sensor_id, time DESC);
+                    CREATE INDEX idx_dns_results_answer_sensor_time ON ts_dns_results_answer (sensor_id, time DESC);
+                    -- CREATE INDEX idx_dns_results_task ON ts_dns_results (task_id);
+                    -- CREATE INDEX idx_dns_results_answer_task ON ts_dns_results_answer (task_id);
+                    --
+                    CREATE INDEX idx_http_results_sensor_time ON ts_http_results (sensor_id, time DESC);
+                    --
+                    CREATE INDEX idx_icmp_results_sensor_time ON ts_icmp_results (sensor_id, time DESC);
+                    --
+                    CREATE INDEX idx_traceroute_results_sensor_time ON ts_traceroute_results (sensor_id, time DESC);
+                    CREATE INDEX idx_traceroute_results_hop_sensor_time ON ts_traceroute_results_hop (sensor_id, time DESC);
+                    -- CREATE INDEX idx_traceroute_results_task ON ts_traceroute_results (task_id);
+                    -- CREATE INDEX idx_traceroute_results_hop_task ON ts_traceroute_results_hop (task_id);`).Error
```

Comment:
@arthureld the indexes here are commented out - do enable them please. Also you should move them in a separate  migration since in the initial one they wouldn't take effect.